README.md

```diff
@@ -65,15 +64,15 @@ arch -x86_64 /usr/local/bin/brew install llvm
 
 ## Usage
 
-Once the executable has been compiled and installed, `tdms` should be in the
-`PATH`.  Check that installation worked with
+Once the executable has been compiled and installed, `tdms` should be in the `PATH`.
+Check that installation worked with
 
 ```bash
 $ tdms -h
 ```
 
-You can invoke it directly or call it from a MATLAB script.  We recommend that
-beginners with MATLAB installed start with the demonstration MATLAB script.
+You can invoke it directly or call it from a MATLAB script.
+We recommend that beginners with MATLAB installed start with the demonstration MATLAB script.
```

Comment:
It has been a while since we checked that the example script actually works... Also with the pending updates to `iteratefdtd_matrix`, we should probably do this before the project ends. Will add this to #247